platform/darwin/src/MGLAttributionInfo.mm

```diff
@@ -126,13 +128,34 @@ - (instancetype)initWithTitle:(NSAttributedString *)title URL:(NSURL *)URL {
 }
 
 - (nullable NSURL *)feedbackURLAtCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate zoomLevel:(double)zoomLevel {
+    return [self feedbackURLForStyleURL:nil atCenterCoordinate:centerCoordinate zoomLevel:zoomLevel direction:0 pitch:0];
+}
+
+- (nullable NSURL *)feedbackURLForStyleURL:(nullable NSURL *)styleURL atCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate zoomLevel:(double)zoomLevel direction:(CLLocationDirection)direction pitch:(CGFloat)pitch {
     if (!self.feedbackLink) {
         return nil;
     }
-
-    NSURLComponents *components = [NSURLComponents componentsWithURL:self.URL resolvingAgainstBaseURL:NO];
-    components.fragment = [NSString stringWithFormat:@"/%.5f/%.5f/%i",
-                           centerCoordinate.longitude, centerCoordinate.latitude, (int)round(zoomLevel + 1)];
+    
+    NSURLComponents *components = [NSURLComponents componentsWithString:@"https://www.mapbox.com/feedback/"];
+    components.fragment = [NSString stringWithFormat:@"/%.5f/%.5f/%.2f/%.1f/%i",
```

Comment:
This format string hard-codes precisions for the various components instead of varying the latitude and longitude precision by the zoom level. Do you think that would be a problem @tristen @mollymerp?